modules/light-clients/08-wasm/keeper/querier_test.go

```diff
@@ -44,19 +44,20 @@ func MockCustomQuerier() func(sdk.Context, json.RawMessage) ([]byte, error) {
 	}
 }
 
-func (suite *TypesTestSuite) TestCustomQuery() {
+func (suite *KeeperTestSuite) TestCustomQuery() {
 	testCases := []struct {
 		name     string
-		malleate func()
+		malleate func(k *keeper.Keeper)
```

Comment:
might be nice to change these tests to use `GetSimApp(suite.chainA).WasmClientKeeper` directly inside malleate instead of passing the arg. If only for consistencies sake with the rest of the codebase.